src/install/common.py

```diff
@@ -113,3 +97,13 @@ def main(distribution):  # pylint: disable=too-many-statements
         Path('start_all_installed_fact_components').symlink_to('src/start_fact.py')
 
     return 0
+
+
+def _update_package_sources(distribution):
+    if distribution == 'fedora':
+        logging.info('Updating system')
```

Comment:
Yeah I know - picky - but ... why keep the code duplication?

Reply:
fixed